test/libsolidity/smtCheckerTests/blockchain_state/library_public_2.sol

```diff
@@ -21,6 +21,6 @@ contract C {
 // SMTIgnoreCex: yes
 // ----
 // Warning 4588: (238-243): Assertion checker does not yet implement this type of function call.
+// Warning 8656: (54-67): CHC: Insufficient funds happens here.
```

Comment:
This is a false postivie. I'm surprised, because there are very similar tests without a false positive.

Reply:
Why do you think this is false positive?

Reply:
To make sure I'll ask on dev channel on element.

Reply:
But it looks like msg.value is reflected in balance just before the execution:https://ethereum.stackexchange.com/questions/29991/when-is-this-balance-updated

Reply:
We overlooked the fact that the library function is public and can be called in `C.l(address)`. This should cause the inefficient funds warning.